### PR TITLE
TO DELETE! added setPage onClick and movement of the arrows

### PR DESCRIPTION
**Pull request by KatarzynaZaloba**

### CODE REVIEW

**Review by AdrianJar — APPROVED** [review]
nicely done, nice that you implemented new stuff into our code!

File: src/feature/People/PeopleList/List/styled.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+export const Tile = styled.div`
+    background-color: ${({ theme }) => theme.colors.white};
+    padding: 16px;
+    width: 208px;
+    height: 339px;
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    border-radius: 5px;
+`;
+
+export const Poster = styled.img`
```
Comment (KatarzynaZaloba) [inline src/feature/People/PeopleList/List/styled.js]: I need your help here - on Figma there is Poster hidden under it's cover. Hard to say... I mean part of the poster is under the white cover. I used z-index -1, but didn't help. Any idea?

Comment (ghost) [reply src/feature/People/PeopleList/List/styled.js]: This is an important question, I will also have the same problem when the film does not have a poster.
We will try to find a solution together

**Review by Marianna-Weychan — APPROVED** [review]
Great job, another brick in our application :) but I understand that `const data` is temporary until you do the logic ?

**Comment by KatarzynaZaloba** [comment]
@AdrianJar I need to to resolve the confilcts before I can merge the branch. Can you look at it in free time?

**Comment by KatarzynaZaloba** [comment]
> Great job, another brick in our application :) but I understand that `const data` is temporary until you do the logic ?

Yes, for now it's only template for API's data

**Pull request closed without merging**